lib/foundation/mixin/after_render_mixin.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:go_router/go_router.dart';
+
+mixin AfterRenderMixin on StatefulWidget {
```

Comment:
[nits]
moveAfterRender の中で　useEffect を利用しているため、HookConsumerWidget　などのほうが適切そうです。StatefulWidget では使えない or 使う想定ではない場合、このように指定すると、後々、混乱させてしまうことになりそうです。

```suggestion
mixin AfterRenderMixin on HookConsumerWidget {
```

Reply:
@blendthink 

レビューありがとうございます！
おっしゃる通り HookConsumerWidget の方が適切ですね💦
ac0e5b9  こちらで対応しましたのでご確認よろしくお願いします🙏